internal/plugins/manifests/init.go

```diff
@@ -97,9 +127,16 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# BUNDLE_IMG defines the image:tag used for the bundle. 
+# IMAGE_TAG_BASE defines the repository, namespace, and part of the image name for remote images.
+# This variable is used to construct full image tags for bundle and catalog images.
+#
+# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
+# quay.io/%[1]s/%[2]s-bundle:$VERSION and quay.io/%[1]s/%[2]s-catalog:$VERSION.
+IMAGE_TAG_BASE ?= quay.io/%[1]s/%[2]s
```

Comment:
Is required to be fixed for quay,io? Cannot users use any other registry?

Reply:
This is not required, it's just a suggestion. You could also do something like `<domain>/<projectName>`, ex. `my.domain/memcached-operator`.